frontend/src/pages/index.js

```diff
@@ -52,30 +85,67 @@ class LoginPage extends Component {
-            <Row className="login-loading-box">
-              <Col md="4" />
-              <Col md="2">
-                <ReactLoading className="login-box" type="spinningBubbles" color="#000" />
-              </Col>
-            </Row>
-          ) : (
-            <div className="align-middle login-box">
-              <h4>Enter Key:</h4>
-              <Input
-                autoFocus={true}
-                type="text"
-                value={this.state.currentKey}
-                onChange={this.onTextChange}
-                name="Input Key"
-                placeholder="Input Your Key"
-                onKeyPress={this._handleKeyPress}
+          <Card className="login-card" style={{ width: '400px', height: '60%' }}>
+            <CardTitle>
+              <h3 style={{ textAlign: 'center', paddingTop: '10px' }}>Login</h3>
+            </CardTitle>
+            <CardBody>
+              <Form>
+                <FormGroup>
+                  <Label for="exampleEmail">Email</Label>
+                  <Input
+                    type="email"
+                    name="email"
+                    id="exampleEmail"
+                    maxLength="64"
+                    value={this.state.email}
+                    onChange={this.handleChange}
+                    required
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="examplePassword">Password</Label>
+                  <Input
+                    type="password"
+                    name="password"
+                    minLength="8"
+                    maxLength="64"
+                    value={this.state.password}
+                    onChange={this.handleChange}
+                    required
+                  />
+                </FormGroup>
+                <Button color="outline-secondary" onClick={this.handleSubmit}>
+                  Submit
+                </Button>
+              </Form>
+
+              <GoogleLogin
+                className="btn sign-in-btn"
+                clientId="992779657352-2te3be0na925rtkt8kt8vc1f8tiph5oh.apps.googleusercontent.com"
+                responseType="id_token"
+                buttonText={this.props.role}
+                scope="https://www.googleapis.com/auth/userinfo.email"
+                onSuccess={this.handleGoogle}
               />
-              <Button className="mt-3" color="primary" onClick={this.handleSubmit}>
-                Login
+            </CardBody>
+          </Card>
+          <Button color="outline-primary" onClick={() => Router.push('/register')}>
+            {"Don't have an account? Register here!"}
+          </Button>
+          <Modal autoFocus={false} isOpen={this.state.showInvalidRequestModal}>
+            <ModalHeader>{'There was an error in your request.'}</ModalHeader>
+            <ModalBody>
+              {this.state.errorMessage
+                ? this.state.errorMessage
+                : 'There was an error in your request'}
```

Comment:
stylistic change, could be the following: 
```js
{this.state.errorMessage || 'There was an error in your request'}
```
Is that something we'd want or should we stick to ternary?

Reply:
Much prefer this way as well